lib/integrations/jira.rb

```diff
@@ -107,6 +109,16 @@ def create_webhook_timeout_issue
     }
   end
 
+  def create_integration_test_issue
+    {
+      project: { key: settings[:project_key] },
+      labels: ['Integration Test'],
+      issuetype: { name: 'Bug' },
```

Comment:
`issuetype` is a field name for the jira API. I don't think we should change that: https://developer.atlassian.com/jiradev/jira-apis/jira-rest-apis/jira-rest-api-tutorials/jira-rest-api-example-create-issue